Setup/BuildInstallers.VS2017.cmd

```diff
@@ -0,0 +1,49 @@
+@echo off
+
+cd /d "%~p0"
+
+IF NOT EXIST "%~p0\tools\vswhere1.0.62.exe" (
+    "%~p0\tools\curl.exe" -L -k -o %~p0\tools\vswhere1.0.62.exe https://github.com/Microsoft/vswhere/releases/download/1.0.62/vswhere.exe -L https://github.com > NUL
```

Comment:
Maybe use name `vswhere.exe`? So that when we update it, the old version does not remain on the disk